docs/integrations/sources/payfit.md

```diff
@@ -0,0 +1,26 @@
+# PayFit
+Connector for PayFit.
```

Comment:
Can you provide a better description about what PayFit is and a link to their website?